filter_test.go

```diff
@@ -370,14 +370,6 @@ func TestFilterFwAddDel(t *testing.T) {
 	if fw.Police.Rate.Rate != filter.Police.Rate.Rate {
 		t.Fatal("Police Rate doesn't match")
 	}
-	for i := range fw.Rtab {
```

Comment:
So this test was passing because the two arrays  `fw.Rtab` and `filter.Rtab` were actually empty because of the bug, correct ?

Reply:
yes